.buildkite/scripts/steps/cloud/build_and_deploy.sh

```diff
@@ -128,7 +128,7 @@ cat << EOF | buildkite-agent annotate --style "info" --context cloud
 
   Elasticsearch: $CLOUD_DEPLOYMENT_ELASTICSEARCH_URL
 
-  Credentials: \`vault read secret/kibana-issues/dev/cloud-deploy/$CLOUD_DEPLOYMENT_NAME\`
+  Credentials: \`vault_get cloud-deploy/$CLOUD_DEPLOYMENT_NAME\`
```

Comment:
This is meant to be copy/pasted and I think asking users to source util.sh will be confusing.  Do you think there's a way we can continue using the `vault read` string?

Reply:
I see! I think for users, there's no change, we'll all probably move to the other vault - and we can always try both vault addresses